libs.elimpl/external/el-impl-2.2.99-license.txt

```diff
@@ -0,0 +1,387 @@
+Name: Expression Language Implementation
+Description: Provides an implementation of the Expression Language (EL).
+Version: 2.2.99 (Implementation version: 3.0-b07)
```

Comment:
Should we remove `(Implementation version: 3.0-b07)`? or use it?

Reply:
This fails verification, as the version string is expected verbatim in the filename. As the name refers to the implementation version, I would just use that.

Reply:
I'll fix to `Version: 3.0-b07`. Thanks!